packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1442,4 +1463,74 @@
     <protected>false</protected>
     <shortDescription>Show less</shortDescription>
   </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionTopic</fullName>
+    <value>Is the answer about the right topic?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Question Topic</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionHallucination</fullName>
+    <value>Is the answer free of hallucinated content?</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Feedback Question Hallucination</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_FeedbackQuestionDocumented</fullName>
+    <value>Can the question be answered by the documentation?</value>
```

Comment:
What is the purpose of this question? 😂  I feel like most users who will ask questions to the model might not check the doc X)

Reply:
It's for subject matter experts to tell us.